tools/src/bin/benchmark-clvm-cost.rs

```diff
@@ -512,6 +512,13 @@ pub fn main() {
             extra: None,
             flags: NESTING_BASE_COST | PER_ARG_COST | PER_BYTE_COST | LARGE_BUFFERS,
         },
+        Operator {
+            opcode: 65,
+            name: "sha256tree (atom)",
+            arg: Placeholder::SingleArg(Some(g1)),
```

Comment:
I don't think `g1` is right here, it looks like it's just going to pass in valid G1 points. I can't recall exactly how this tool works though.

Reply:
maybe richard's suggestion is simpler. However, traversing the tree (probably) has a cost as well, not just sha256.